addons/payment_razorpay/models/payment_transaction.py

```diff
@@ -49,6 +50,19 @@ def _get_specific_rendering_values(self, processing_values):
         converted_amount = payment_utils.to_minor_currency_units(self.amount, self.currency_id)
         base_url = self.provider_id.get_base_url()
         return_url_params = {'reference': self.reference}
+
+        phone = self.partner_phone
+        error_message = _("Partner's phone number is missing.")
+        if self.partner_phone:
+            # sanitize partner phone
+            country_code = self.partner_country_id.code
+            country_phone_code = self.partner_country_id.phone_code
+            phone_info = phone_sanitize_numbers([self.partner_phone], country_code, country_phone_code)
```

Comment:
And this to keep the line under 100 chars
```suggestion
            phone_info = phone_sanitize_numbers([phone], country_code, country_phone_code)
```